server/game/cards/11-TCaR/EpitaphBranchOffice.js

```diff
@@ -0,0 +1,36 @@
+const DeedCard = require('../../deedcard.js');
+
+class EpitaphBranchOffice extends DeedCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            title: 'Epitaph Branch Office',
+            cost: ability.costs.bootSelf(),
+            when: {
+                onSundownAfterVictoryCheck: () => this.game.findCardsInPlay(card =>
+                    card.location === 'play area' && card.controller === this.controller && card.getType() === 'dude' &&
+                        card.control === 0 && card.influence >= 1 && card.isInOpponentsHome()).length > 0
+            },
+            handler: context => {
+                this.game.promptForSelect(context.player, {
```

Comment:
When selecting another target in the card script, we should use `selectAnotherTarget`.
```suggestion
                context.ability.selectAnotherTarget(context.player, context, {
```